diff --git a/examples/getting_started.cpp b/examples/getting_started.cpp
index 83bb10d5..aa094dd0 100644
--- a/examples/getting_started.cpp
+++ b/examples/getting_started.cpp
@@ -432,7 +432,7 @@ void getting_started_tutorial(engine::kind engine_kind) {
     // memory format.
     std::vector<float> relu_image(image_size);
     read_from_dnnl_memory(relu_image.data(), dst_mem);
-    /*
+
     // Check the results
     for (int n = 0; n < N; ++n)
         for (int h = 0; h < H; ++h)
@@ -452,7 +452,6 @@ void getting_started_tutorial(engine::kind engine_kind) {
                     }
                 }
     // [Check the results]
-    */
 }
 
 /// @page getting_started_cpp
